src/components/Header/Header.module.css

```diff
@@ -1,20 +1,137 @@
 .header {
-    background-color: #f8f9fa;
-    padding: 10px 20px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  width: 100%;
+  background-color: var(--dark-theme-background);
+  padding-left: 10%;
+  padding-right: 10%;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+.logo {
+  width: 100px;
+}
+.header-nav {
+  width: 1000px;
+  max-width: 750px;
+  margin-right: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.nav-list {
+  display: flex;
+  align-items: center;
+}
+
+.nav-list-item{
+  list-style: none;
+  color: var(--primary-white);
+  padding-right: 30px;
```

Comment:
Remove the `padding-right`. You were looking for `margin-right` (space outside the element) but since we're using flex, we can use the `gap` property on the parent (`.nav-list` in this case) to easily set space _only_ between elements and not add extra spacing on the end (the right end, in this case).